### PR TITLE
fix refresh, removed unused code

### LINKED ISSUE #88: EntityManger's factories
The EntityManger has two factories: `EntityManger::create` and `EntityManger::buildWithHost`. 

It seams like

```php
EntityManger::buildWithHost($host);
// same as
EntityManger::create($host);
```

Is that correct? If so, we should break BC while we can (before stable release) and remove `EntityManger::buildWithHost`.

### LINKED ISSUE #89: ObjectHydration does not exist
See here: https://github.com/graphaware/neo4j-php-ogm/blob/1.0.0-beta22/src/EntityManager.php#L255

There is no object called `ObjectHydration` anymore. 
The getHyrdrator function is used by the `EnittyPersister::refresh`

### LINKED ISSUE #94: The Collection::addInit is never used
Im not sure we need that function. I suggest removing it.

### LINKED ISSUE #95: Variable not used
This is some strange looking code. Why is `$el` never used?

https://github.com/graphaware/neo4j-php-ogm/blob/1.0.0-beta22/src/Metadata/RelationshipMetadata.php#L255

### PR DESCRIPTION
**Pull request by ikwattro**
Fixes #95 

Fixes #94 

Fixes #89 

Fixes #88

### CODE REVIEW

**Comment by ikwattro** [comment]
NB: Don't merge yet. I want to add a test for refresh();

File: src/Hydrator/EntityHydrator.php [hunk 1]
```diff
@@ -226,6 +226,17 @@ protected function hydrateNode(Node $node, $class = null)
...
 
+    /**
+     * @param Node $node
+     * @param object $entity
+     */
+    public function refresh(Node $node, $entity)
+    {
+        $this->hydrateProperties($entity, $node);
+        $this->hydrateLabels($entity, $node);
+
```
Comment (kbsali) [inline src/Hydrator/EntityHydrator.php]: cs fixer would fix this ;)

Comment (ikwattro) [inline src/Hydrator/EntityHydrator.php]: @kbsali I tried the cs fixer config locally. Impossible to run it with php7. I'll try to have a proper config compatible with my setup when I have some time

**Comment by Nyholm** [comment]
👍

**Pull request merged**